array_api_compat/torch/_aliases.py

```diff
@@ -9,14 +9,15 @@
                                vecdot as _aliases_vecdot)
 from .._internal import get_xp
 
+import torch
+
 from typing import TYPE_CHECKING
 if TYPE_CHECKING:
     from typing import List, Optional, Sequence, Tuple, Union
     from ..common._typing import Device
     from torch import dtype as Dtype
 
-import torch
-array = torch.Tensor
+    array = torch.Tensor
```

Comment:
I'll be honest, I haven't actually checked any of the type hints that are here in any way. I've just copied them from the spec, and put these basic imports here so that pyflakes is quiet about it. If/when someone actually complains about proper mypy support, we can fix things up here so that they actually work. 

As far as the symbol being defined, the `from __future__ import annotations` at the top of the file prevents the annotations from actually being executed at runtime.